controllers/ingress/group_controller.go

```diff
@@ -63,6 +66,7 @@ func NewGroupReconciler(cloud aws.Cloud, k8sClient client.Client, eventRecorder
 // GroupReconciler reconciles a ingress group
 type groupReconciler struct {
 	k8sClient        client.Client
+	config           config.IngressConfig
```

Comment:
seems this member is not used, we don't need to save it as member then.

Reply:
It is in use for MaxConcurrentReconciles. I will rename it to ingressConfig